content/en/docs/concepts/workloads/pods/pod-lifecycle.md

```diff
@@ -534,10 +534,11 @@ processing its startup data, you might prefer a readiness probe.
 
 {{< note >}}
 If you want to be able to drain requests when the Pod is deleted, you do not
-necessarily need a readiness probe; on deletion, the Pod automatically puts itself
-into an unready state regardless of whether the readiness probe exists.
-The Pod remains in the unready state while it waits for the containers in the Pod
-to stop.
+necessarily need a readiness probe; when the Pod is deleted, the corresponding endpoint
+will update its [conditions](../../services-networking/endpoint-slices.md#conditions).
```

Comment:
```suggestion
will update its [conditions](/docs/concepts/services-networking/endpoint-slices/#conditions).
```
See https://kubernetes.io/docs/contribute/new-content/open-a-pr/#preview-locally for how to preview your changes locally.

Reply:
Thank you!